.github/workflows/ci.yml

```
@@ -192,6 +192,22 @@ jobs:
           name: nuget-packages
           path: ./Bin/Release/ILGPU*.${{ steps.package.outputs.version }}.*nupkg
 
+  release:
+    if: startsWith(github.ref, 'refs/tags/v')
+    needs: [check-style, build-and-test]
```

ISSUE: Missing reference to `package`.
```suggestion
    needs: [check-style, build-and-test, package]
```